docs/jwst/jump/description.rst

```diff
@@ -107,12 +40,38 @@ counts. These excess counts are, in general, below the detection
 threshold of normal cosmic rays.
 
 To constrain the effect of this halo, the jump step will fit ellipses or circles that
-enclose the large events and expand the ellipses and circles by the input expansion_factor
-and mark them as jump.
+enclose the large events and expand the ellipses and circles by the input
+`expand_factor` and mark them as jump (see :ref:`jump step arguments <jump_arguments>`
+for details).
 
-The two types of detectors respond differently. The large events in the near-infrared
+The two different types of JWST detectors respond differently. The large events in the near-infrared
 detectors are almost always circles with a central region that is saturated.
 The saturated core allows the search for smaller events without false positives.
-The MIRI detectors do not, in general, have a saturated center and are only rarely circular.
+The mid-IR (MIRI) detectors do not, in general, have a saturated center and are only rarely circular.
 Thus, we fit the minimum enclosing ellipse and do not require that there are saturated pixels
 within the ellipse.
+
+Multiprocessing
+---------------
+This step has the option of running in multiprocessing mode. In that mode it will
+split the input data cube into a number of row slices based on the number of available
+cores on the host computer and the value of the ``max_cores`` input parameter. By
+default the step runs on a single processor. At the other extreme, if ``max_cores`` is
+set to "all", it will use all available cores (real and virtual). Testing has shown
+a reduction in the elapsed time for the step proportional to the number of real
+cores used. Using the virtual cores also reduces the elapsed time, but at a slightly
+lower rate than the real cores.
+
+If multiprocessing is requested, the input cube will be divided into a number of
+slices in the row dimension (with the last slice being slightly larger, if needed),
+and sent for processing in parallel.
```

Comment:
It should be noted that it's possible for the requested number of slices to be greater than the number of rows.  In this case, the number of slices used is set to the number of rows.